v4-client-py/v4_client_py/clients/helpers/chain_helpers.py

```diff
@@ -20,15 +20,17 @@ class OrderTimeInForce(Flag):
     FOK = auto()    # Fill or Kill
 
 class OrderExecution(Flag):
```

Comment:
This should just be removed since it's confusing people, detailed reasons in this ticket [here](https://linear.app/dydx/issue/TRCL-2707/remove-orderexecution-in-favor-of-ordertimeinforce-use-separate-method).